component/prometheus/exporter/redis/redis.go

```diff
@@ -60,18 +61,21 @@ type Config struct {
 	CheckStreams            []string          `river:"check_streams,attr,optional"`
 	CheckSingleStreams      []string          `river:"check_single_streams,attr,optional"`
 	CountKeys               []string          `river:"count_keys,attr,optional"`
-	ScriptPath              string            `river:"script_path,attr,optional"`
+	ScriptPaths             []string          `river:"script_paths,attr,optional"`
```

Comment:
This is technically a breaking change, so we should be a little considerate here.

Do you want it to be a breaking change? If so, that's probably fine pre-v1. If not, you might have to consider a different route (like changing the documentation for `script_path` to mention delimiting multiple scripts by comma, or supporting both `script_path` and `script_paths` but have them be mutually exclusive)

Reply:
It should probably be left as a non-breaking change. I like the idea of having both options, are there any existing flow components you know of that have a pattern like that?

Reply:
> I like the idea of having both options, are there any existing flow components you know of that have a pattern like that?

There's a few fields in the squashed HTTPClientConfig blocks which are mutually exclusive, you could check there